src/Data/Semigroup/Foldable.purs

```diff
@@ -118,9 +120,15 @@ sequence1_ = traverse1_ identity
 maximum :: forall f a. Ord a => Foldable1 f => f a -> a
 maximum = ala Max foldMap1
 
+maximumBy :: forall f a. Ord a => Foldable1 f => (a -> a -> Ordering) -> f a -> a
```

Comment:
The Ord constraint is redundant here, right?

Reply:
I forgot to remove to remove the constraint after copying Data.Foldable.maximumBy and Data.Foldable.minimumBy 🤦